app/build.gradle

```diff
@@ -1,14 +1,14 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
+    compileSdkVersion 28
     defaultConfig {
         applicationId "com.pixlee.pixleeandroidsdk"
         minSdkVersion 15
-        targetSdkVersion 25
+        targetSdkVersion 28
```

Comment:
Developers working for the latest SDK should specify the version name there so that the IDE tells you what can go wrong the specified SDK version. You can also get hint about what will happen on lower versions.

More information: https://www.thedroidsonroids.com/blog/android/compile-min-max-and-target-sdk-versions